use-cases/demos/classification/tensorflow-keras/entrypoint.sh

```diff
@@ -9,7 +9,12 @@
 cid=$(date +%Y%m%d%H%M%S%N)
 echo "cid: $cid"
 
-PROFILE_NAME="demo_tensorflow_keras_classification"
-
-python3 classification/tensorflow-keras/tensorflow_keras_classification.py \
--m "$MQTT_HOSTNAME" -p "$MQTT_PORT" -t "$MQTT_TOPIC" -i "$INPUT_SRC" | tee >/tmp/results/r"$cid"_"$PROFILE_NAME".jsonl >(stdbuf -oL sed -n -e 's/^.*FPS: //p' | stdbuf -oL cut -d , -f 1 > /tmp/results/pipeline"$cid"_"$PROFILE_NAME".log)
+if [ $USE_TFLITE -eq 1 ]; then
+    PROFILE_NAME="demo_tensorflow_lite_keras_classification"
+    DETECTION_SCRIPT="classification/tensorflow-keras/liteRT_keras_classification.py"
+else
+    PROFILE_NAME="demo_tensorflow_keras_classification"
+    DETECTION_SCRIPT="classification/tensorflow-keras/tensorflow_keras_classification.py"
```

Comment:
this is ok, just FYI- i usually prefer to have default value set first and then just overwrite the value if USE_TFLITE == 1